### PR TITLE
feature/SRE-674 chart minor version increase, min EKS version

### PR DESCRIPTION
**Pull request by andrzej-tricentis**

### CODE REVIEW

**Review by byangtri — APPROVED** [review]

File: Charts/qtest-mgr/Chart.yaml [hunk 1]
```diff
@@ -65,7 +65,7 @@ icon: https://images.g2crowd.com/uploads/product/image/social_landscape/social_l
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
 
-version: 1.4.2
+version: 1.5.2
```
Comment (pavel-spacil) [inline Charts/qtest-mgr/Chart.yaml:68]:
```suggestion
version: 1.5.0
```

File: Charts/qtest-parameters/Chart.yaml [hunk 2]
```diff
@@ -25,11 +25,11 @@ maintainers:
   - name: tomasw
     email: t.wallick@ext.tricentis.com
 home: https://github.com/Tricentis-qTest/qtest-chart.git
-kubeVersion: ">=1.18.0-0"
+kubeVersion: ">=1.24.0-0"
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.0.12
+version: 1.1.12
```
Comment (pavel-spacil) [inline Charts/qtest-parameters/Chart.yaml:32]:
```suggestion
version: 1.1.0
```

File: Charts/qtest-pulse/Chart.yaml [hunk 3]
```diff
@@ -25,11 +25,11 @@ maintainers:
   - name: tomasw
     email: t.wallick@ext.tricentis.com
 home: https://github.com/Tricentis-qTest/qtest-chart.git
-kubeVersion: ">=1.18.0-0"
+kubeVersion: ">=1.24.0-0"
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.0.12
+version: 1.1.12
```
Comment (pavel-spacil) [inline Charts/qtest-pulse/Chart.yaml:32]:
```suggestion
version: 1.1.0
```

File: Charts/qtest-scenario/Chart.yaml [hunk 4]
```diff
@@ -25,12 +25,12 @@ maintainers:
...
     email: t.wallick@ext.tricentis.com
 home: https://github.com/Tricentis-qTest/qtest-chart.git
-kubeVersion: ">=1.18.0-0"
+kubeVersion: ">=1.24.0-0"
 icon: https://images.g2crowd.com/uploads/product/image/social_landscape/social_landscape_c2d6524c36c94138504f7a18bf179d76/tricentis-qtest.png
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.0.14
+version: 1.1.14
```
Comment (pavel-spacil) [inline Charts/qtest-scenario/Chart.yaml:33]:
```suggestion
version: 1.1.0
```

File: Charts/qtest-session/Chart.yaml [hunk 5]
```diff
@@ -23,11 +23,11 @@ maintainers:
   - name: liemng
     email: l.nguyen@tricentis.com
 home: https://github.com/Tricentis-qTest/qtest-chart.git
-kubeVersion: ">=1.18.0-0"
+kubeVersion: ">=1.24.0-0"
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.2.1
+version: 1.3.1
```
Comment (pavel-spacil) [inline Charts/qtest-session/Chart.yaml:30]:
```suggestion
version: 1.3.0
```

**Review by pavel-spacil — COMMENTED** [review]
Just do the semver right 😉

**Review by pavel-spacil — APPROVED** [review]

**Pull request merged**